puzzle/models/sql/models.py

```diff
@@ -27,11 +27,18 @@ class Case(BASE):
     pedigree = Column(String) # For storing madeline info
     compressed=Column(Boolean, default=False)
     tabix_index=Column(Boolean, default=False)
-    
+
     created_at = Column(DateTime, default=datetime.now)
     updated_at = Column(DateTime, default=datetime.now, onupdate=datetime.now)
 
     @property
+    def case_comments(self):
+        """Return only comments made on the case."""
+        comments = (comment for comment in self.comments if
+                    comment.variant_id is None)
```

Comment:
does this mean that if `variant_id` is `None`, then the comment belongs to this case? so this is mutually exclusive? Can a comment belong to more than one case?